src/wiseagents/vectordb/lang_chain_wise_agent_vector_db.py

```diff
@@ -20,7 +20,9 @@ def __new__(cls, *args, **kwargs):
         """Create a new instance of the class, setting default values for the instance variables."""
         obj = super().__new__(cls)
         obj._embedding_model_name = DEFAULT_EMBEDDING_MODEL_NAME
-        obj._embedding_function = HuggingFaceEmbeddings(model_name=DEFAULT_EMBEDDING_MODEL_NAME)
+        model_kwargs = dict()
+        model_kwargs['tokenizer_kwargs']={"clean_up_tokenization_spaces": True}
```

Comment:
Shouldn't this be False too?

Reply:
The warning says: clean_up_tokenization_spaces was not set. It will be set to True by default. This behavior will be depracted in transformers v4.45, and will be then set to False by default.

Reply:
Ok that's fine but we should use the same default value in both the new method and the init method. Right now, True is being used for new and False is being used in the init method. So we should just make it consistent everywhere.

(The new method gets called when instantiating things defined in the YAML configuration whereas init gets called when instantiating something explicitly. We should use the same default in both places).